README.rst

```diff
@@ -71,7 +71,20 @@ To check if everything is ready to go, you can always test both fixtures:
 Configuration
 =============
 
-You can define your own path for elasticsearch logs directory. There are three ways to achieve this:
+You can define your own paths and/or variables for elasticsearch:
+
+    * logs directory (logsdir, elasticsearch_logsdir, --elasticsearch-logsdir),
```

Comment:
it's hard to tell how can I use these variables or where. Maybe using http://docutils.sourceforge.net/docs/user/rst/quickref.html#tables would help here?